functions-and-scope.js

```diff
@@ -15,7 +15,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6
-
+//stap 1: loop door iedere waarde in de array, en controleer of de waarde hoger is of gelijk is aan 8
+//stap 2: gebruik daarbij de lengte van de array als conditie
+//stap 3: return de uitkomst
+let number = 0;
+for (let i = 0; i < grades.length; i++){
+    if(grades[i] >= 8) {
+        number = number + 1;
```

Comment:
Dit kun je versimpelen naar
```suggestion
        number++;
```